def testFail(self):
    assert False
